### DISCUSSION TITLE
How does the "configure grid layout" part in the complex example work?

### DISCUSSION THREAD

**Discussion by TechPro424** [TomSchimansky/CustomTkinter#2057] [Q&A]
How does the "configure grid layout" part in the complex example work? Shouldn't columns 2 and 3 be configured to have a weight of 11?

**Comment by avalon60** [comment]
Not sure where 11 comes from.

The relative weights are proportionate. E.g if you want one of two columns to expand to take a third of the space, you would assign a value of 1 to one column and a value of 2 to the other.

**Reply by TechPro424** [reply]
oh sorry I meant a weight of 1
it was a typo
